src/main/java/com/minecolonies/coremod/util/LanguageHandler.java

```diff
@@ -33,17 +35,84 @@ private LanguageHandler()
      */
     public static void sendPlayerLocalizedMessage(@NotNull final EntityPlayer player, final String key, final Object... args)
     {
-        sendPlayerMessage(player, format(key, args));
+        sendPlayerMessage(player, key, args);
     }
 
     /**
      * Send a message to the player.
      * @param player the player to send to.
+     * @param key the key of the message.
      * @param message the message to send.
      */
-    public static void sendPlayerMessage(@NotNull final EntityPlayer player, final String message)
+    public static void sendPlayerMessage(@NotNull final EntityPlayer player, final String key, final Object... message)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "sendPlayerMessage" is 16 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)